bedrock/contentful/api.py

```diff
@@ -444,54 +450,102 @@ def page(self):
             },
         )
 
-    def render_rich_text(self, node):
+    def render_rich_text(self, node) -> str:
         return self._renderer.render(node) if node else ""
 
-    def get_info_data(self, entry_obj):
+    def _get_preview_image_from_fields(self, fields: dict) -> Union[str, None]:
+        if "preview_image" in fields:
+            # TODO request proper size image
+            preview_image_url = fields["preview_image"].fields().get("file").get("url")
+            return "https:" + preview_image_url
```

Comment:
In my last project we moved away from string concatenation and leaned more heavily on formatted strings, like `f"https:{preview_image_url}"`. Curious how y'all feel or if there's any pattern already in place in bedrock?

Reply:
Yeah, let's go with f strings. This was a lift and shift of existing code, but I should have refactored at the time